tests/brevitas/graph/test_transforms.py

```diff
@@ -287,6 +287,6 @@ def forward(self, x):
     model = TestModel()
     assert model.conv.stride == (1, 1)
 
-    kwargs = {'stride': lambda module: 2 if module.in_channels == 3 else 1}
+    kwargs = {'stride': lambda module, name: 2 if module.in_channels == 3 else 1}
```

Comment:
Check if anything in `brevitas_examples` will be affected by this change.

Reply:
Lambdas gets inspected to decide what's the correct signature to use, so we have backward compatibility